chainstate/tx-verifier/src/transaction_verifier/input_output_policy.rs

```diff
@@ -494,10 +497,10 @@ mod tests {
     #[rstest]
     #[trace]
     #[case(Seed::from_entropy())]
-    fn tx_one_to_many_invalid_outputs(#[case] seed: Seed) {
+    fn tx_one_to_any_invalid_outputs(#[case] seed: Seed) {
```

Comment:
Those seeds pass. 
But I remember @TheQuantumPhysicist was against including particular seeds in tests, even as regression cases.

Reply:
Adding seeds is virtually useless, because any change in the test will cause a chain effect that will change all the cases that come after it, which will result in an accumulation of useless values that are potentially outdated.

It's a nice idea and this is what proptest does because it controls the parameters, but it's not appropriate here, unfortunately. Proptest can do this because the parametrization is not done with the seed, but with the parameters themselves, which cost the formatting issue we complain about due to using a macro.

So, in good faith, we believe that the dev fixing the issue has fixed that problem for the given seed values, and we move on.